### DISCUSSION TITLE
New Releases view is harder to read and visually seems more cluttered

### DISCUSSION THREAD

**Discussion by Malmik** [community/community#6193] [Other Feature Feedback, Questions, & Ideas] [answered]
As i see it, additional border around items under the "Assets" list is unnecessary.
It does not enchance user experience in any way and even seems a bit off. Probably removing it and shifting items from list would be way to go. 

As a side note, blue hyperlink on a version name helped a lot if I were to find specific version by scrolling Releases page. I think removing it made page a lot more clean and better looking, although maby having it in bold would be the best of both worlds.

**Comment by MylesBorins** [comment, marked as answer]
Thank you for bringing your feedback. I think that we are going to have to agree to disagree on this one. We worked closely with design and did user research to come up with the current design and are not planning to make significant changes to it at this time.
